.github/scripts/e2e-failure-triage/triage.py

```
@@ -0,0 +1,1041 @@
+        print("\nNo actionable component failures found.")
+    else:
+        print(f"\nActionable component failures: {len(result.component_failures)}")
+        unmapped = []
+        for cf in result.component_failures:
+            mapping = result.component_mapping.get(cf.component)
+            print(f"\n  Component:      {cf.component}")
+            print(f"  Test:           {cf.test_name}")
+            if mapping and mapping.jira_component:
+                print(f"  Jira component: {mapping.jira_component}")
+            else:
+                print(f"  Jira component: NOT MAPPED")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)